.github/workflows/release.yml

```diff
@@ -58,6 +58,9 @@ jobs:
         uses: docker/metadata-action@v5
         with:
           images: ghcr.io/${{ github.repository }}
+          tags: |
+            type=semver,pattern={{version}}
+            type=semver,pattern={{major}}.{{minor}}
```

Comment:
Marvin images tags are currently prefixed with "v": `ghcr.io/undistro/marvin:v0.2.3`
This change would be changing this pattern.